clients/cli/build.rs

```diff
@@ -4,20 +4,31 @@ use std::fs;
 use std::process::Command;
 use std::{env, path::Path};
 
+/// Compiles the protobuf files into Rust code using prost-build.
 fn main() -> Result<(), Box<dyn Error>> {
-    // Skip proto compilation unless build_proto feature is enabled
+    // Skip proto compilation unless build_proto feature is enabled.
     if !cfg!(feature = "build_proto") {
-        println!("Skipping proto compilation (enable with --features build_proto)");
+        println!("cargo:warning=Skipping proto compilation. Enable with `cargo clean && cargo build --features build_proto`");
```

Comment:
The prefix "cargo:warning" ensures that this is printed, and with a nice yellow "Warning"